scripts/windows/restart-helpers.ps1

```diff
@@ -0,0 +1,53 @@
+# //////////////////////////////////////////////////////////////////////
+# restart-helpers.ps1
+
+function Test-Key([string] $path, [string] $key)
+{
+  return ((Test-Path $path) -and ((Get-Key $path $key) -ne $null))
+}
+
+function Remove-Key([string] $path, [string] $key)
+{
+  Remove-ItemProperty -path $path -name $key
+}
+
+function Set-Key([string] $path, [string] $key, [string] $value)
+{
+  Set-ItemProperty -path $path -name $key -value $value
+}
+
+function Get-Key([string] $path, [string] $key)
+{
+  return (Get-ItemProperty $path).$key
+}
+
+
+$global:restartKey = "Restart-And-Resume"
+function Clear-Any-Restart([string] $key=$global:restartKey)
+{
+  if (Test-Key $global:RegRunKey $key) {
+    Remove-Key $global:RegRunKey $key
+  }
+}
+
+$global:RegRunKey ="HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Run"
```

Comment:
Shouldn't we use `RunOnce` instead, to make sure it gets cleaned up automatically?

Reply:
I first went with `RunOnce` and it didn't work at all. But let me retry it now that I can tweak a working system.

Reply:
*Now* it works